site/docs/plugins/inline-query.md

```diff
@@ -0,0 +1,275 @@
+To do this, they start a message with `@your_bot_name` and choose one of the results.
+
+> Revisit the Inline mode section in the [Telegram Bot Features](https://core.telegram.org/bots/features#inline-requests) written by the Telegram team.
+> Further resources are their [detailed description](https://core.telegram.org/bots/inline) of inline bots, as well as the [original blog post](https://telegram.org/blog/inline-bots) announcing the feature, and the Inline mode section in the [Telegram Bot API Reference](https://core.telegram.org/bots/api#inline-mode).
+> They are all worth a read before implementing inline queries for your bot, as inline queries are a little advanced.
+> If you do not feel like reading all of that, then rest assured that this page will walk you through every step.
+
+## Enabling Inline Mode
+
+By default, support for inline mode is disabled for your bot.
+You must contact [@BotFather](https://t.me/BotFather) and enable inline mode for your bot to start receiving inline queries.
+
+Got it?
+Your Telegram client should now display "..." when you type the bot name in any text field, and show a loading spinner.
+You can already start typing something.
+Let us now see how your bot can handle these queries.
+
+## Handling Inline Queries
+
+Once a user triggers an inline query, i.e. starts a message by typing "@your_bot_name ..." in the text input field, your bot will receive updates about this.
+grammY has special support for handling inline queries via the `bot.inlineQuery()` method, as documented on the `Composer` class in the [grammY API Reference](https://deno.land/x/grammy/mod.ts?s=Composer#method_inlineQuery_0).
+It allows you to listen for specific inline queries that match strings or regular expressions.
+If you want to handle all inline queries generically, use `bot.on("inline_query")`.
+
+```ts
+// Listen for specific strings or regular expressions.
+bot.inlineQuery(/best bot (framework|library)/, async (ctx) => {
+  const match = ctx.match; // regex match object
+  const query = ctx.inlineQuery.query; // query string
+});
+
+// Listen for any inline query
+bot.on("inline_query", async (ctx) => {
+  const query = ctx.inlineQuery.query; // query string
+});
+```
+
+Now that we know how to listen for inline query updates, we can answer them with a result list.
+
+## Building Inline Query Results
+
+Building result lists for inline queries is a tedious task because you need to construct [complex nested objects](https://core.telegram.org/bots/api#inlinequeryresult) with a variety of properties.
+Fortunately, you are using grammY, and of course there are helpers that make this task very simple.
+
+Every result needs three things.
+
+1. A unique string identifier.
+2. A _result object_ that describes how to display the inline query result.
+   It can contain things like a title, a link, or an image.
+3. A _message content_ object that describes the content of the message which will be sent by the user if they pick this result.
```

Comment:
Is it true that here `object' should be unformatted, while in the second paragraph `object' is formatted?
```suggestion
3. A _message content object_ that describes the content of the message which will be sent by the user if they pick this result.
```